docs/dev/table/functions.md

```diff
@@ -1666,6 +1678,18 @@ NUMERIC.tan()
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight java %}
+NUMERIC.tanh()
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the hyperbolic tangent of <i>NUMERIC</i>.</p> 
```

Comment:
here?

Reply:
According to the agreement of @xccui  before, it should be no problem, it is based on this line: 

```
NUMERIC.tanh()
```